bin/varnishd/cache/cache_req_body.c

```diff
@@ -122,6 +125,8 @@ vrb_pull(struct req *req, ssize_t maxsize, objiterate_f *func, void *priv)
 			req_bodybytes += l;
 			if (yet >= l)
 				yet -= l;
+			else if (yet > 0)
+				yet = 0;
```

Comment:
Dead branch spotted by @nigoroll.

Reply:
`l` may be increased by `VFP_GetStorage()` and become larger than `yet`, in which case this branch may be reached when the remaining bytes fit entirely in the allocated buffer.

Reply:
Oh, sorry, I confused the `l` for a `1`.